configure

```diff
@@ -16097,20 +16083,28 @@ printf %s "checking for _POSIX_THREADS in unistd.h... " >&6; }
 /* end confdefs.h.  */
 
 #include <unistd.h>
-#ifdef _POSIX_THREADS
-yes
+
+#if defined(_POSIX_THREADS)
+/* pass */
+#else
+#error "check failed"
 #endif
 
+int
+main (void)
+{
+
+  ;
+  return 0;
+}
```

Comment:
Could this noise be reduced by using AC_LANG_SOURCE rather than AC_LANG_PROGRAM?

Reply:
Definitely! Good catch.